Fix for special characters in file name

d4n13l3 opened the pull request:
See issue #9

(I'm a python noob but with a little search i managed to solve this.
It's working but feel free to implement it better if there's a correct
way.)

d4n13l3 committed:
Fix for special characters in file name

See issue #9

(I'm a python noob but with a little search i managed to solve this.
It's working but feel free to implement it better if there's a correct
way.)

cl1ck commented:
hey d4n13l3

thanks for the quick fix! Should work fine, I have some minor remarks though:
1. you already filter out all non alphanumeric characters, so the replace function is redundant
2. I think only allowing alphanumeric characters in the regex is too restrictive. Here is a list of all characters which are not allowed in Windows filenames: http://msdn.microsoft.com/en-us/library/aa365247%28VS.85%29 I'd suggest you just blacklist those.

d4n13l3 committed:
Alphanumerics chars. Redundant function.

Removed the redundant replace function.
Added more valid alphanumeric characters for the video name.

d4n13l3 committed:
Changed Regex #2
